.travis.yml

```diff
@@ -11,13 +11,13 @@ git:
 before_install:
   - source scripts/website/setup-build-tools.sh
 sudo: required
+services:
+  - docker
 stages:
   - name: test
   - name: release
     if: (branch = master AND type = push) OR (tag IS present)
-  - name: upload-launcher
```

Comment:
Why remove the `upload-launcher` stage? Couldn't it be kept along the new `upload-artifacts`?

Reply:
Because, if my understanding of travis is correct, jobs run only in parallel if they're in [the same stage](https://docs.travis-ci.com/user/build-stages/#how-do-build-stages-work).

Reply:
I've put the `upload-launcher` stage back in as it was before.